main.bal

```diff
@@ -334,8 +334,9 @@ service / on new http:Listener(serverPort) {
     #
     # + page - the requested page (starting with page 0)
     # + 'item\-count - the number of items per page (5 <= item-count <= 500)
+    # + ascending - 1 for asc sort, 0 for desc sort by step sequence
```

Comment:
This should either be boolean or `1` vs. `-1`. Then positive is ascending, negative is descending.
Also rename to something like `sort` or `order`.